go.mod

```diff
@@ -1,70 +1,77 @@
 module github.com/restic/restic
 
-go 1.23
+go 1.23.0
```

Comment:
The extra zero is necessary to prevent `go mod tidy` from automatically inserting a `gotoolchain` line.